### ISSUE TITLE
[question] Acrobot-v1 DQN

### ISSUE THREAD

**Issue by HakarK** [hill-a/stable-baselines#687]
The acrobot won't stabilize at the equilibrium, it just keeps on spinning around. Why is that? The code can be seen under.

> import gym
> 
> from gym import spaces
> from AcrobotEnv import AcrobotEnv
> import tensorflow as tf
> 
> from stable_baselines import DQN
> from stable_baselines.deepq.policies import MlpPolicy
> from stable_baselines.common.env_checker import check_env
> env=AcrobotEnv()
> check_env(env, warn=True)
> model = DQN('MlpPolicy', env, verbose=1).learn(10000)
> 
> obs = env.reset()
> for i in range(1000):
>     action, _states = model.predict(obs)
>     obs, rewards, dones, info = env.step(action)
>     print(obs)
> 
>     env.render()

**Comment by Miffyli** [comment]
Try with the hyperparameters from [rl-baselines-zoo](https://github.com/araffin/rl-baselines-zoo/blob/master/hyperparams/dqn.yml#L41). Some of the environment-algorithm combinations are sensitive to these.

**Comment by araffin** [comment]
As @Miffyli says and as written in the [doc](https://stable-baselines.readthedocs.io/en/master/guide/rl_tips.html):
- "Therefore, we highly recommend you to take a look at the RL zoo (or the original papers) for tuned hyperparameters."
- "For better performance, increase the training budget"

The number of training timesteps you used is quite small.

**Comment by HakarK** [comment]
> Try with the hyperparameters from [rl-baselines-zoo](https://github.com/araffin/rl-baselines-zoo/blob/master/hyperparams/dqn.yml#L41). Some of the environment-algorithm combinations are sensitive to these.

Where do I change these parameters? and I'm assuming the customDQNPolicy is:
class CustomDQNPolicy(FeedForwardPolicy):
    def __init__(self, *args, **kwargs):
        super(CustomDQNPolicy, self).__init__(*args, **kwargs,
                                           layers=[32, 32],
                                           layer_norm=False,
                                           feature_extraction="mlp")

**Comment by araffin** [comment]
> Where do I change these parameters? 

Please read the documentation first, all your answers are there.

**Comment by HakarK** [comment]
> Please read the documentation first, all your answers are there.
Okei, thanks guys:)

**Comment by HakarK** [comment]
I have changed the hyperparameters and implemented CustomDQNPolicy, but the acrobot keeps on spinning.

**Comment by araffin** [comment]
As mentioned before the acrobot is not a pendulum, the goal is to reach a target height, not to balance it.
What is your mean episode reward?

**Comment by HakarK** [comment]
The mean episode reward is -91.

**Comment by araffin** [comment]
>The mean episode reward is -91.

This corresponds to the results achieved in the zoo, and looking at the policy, it solves the task.
